README.md

```diff
@@ -93,7 +93,6 @@ apis:
                         type: 'http'
                         urls:
                             - 'http://www.mocky.io/v2/5cbd4d1d2f0000e70a16cc0e'
-                    throttlePolicy: 10kPerMin
```

Comment:
Correct attribute should be `throttle_policy`
@nuwanbando Can you please change `throttlePolicy` to `throttle_policy`. That should work